assignments/closure.js

```diff
@@ -1,20 +1,53 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
-
+const name = "Matt";
+function whatsUrName(){
+  return name;
+}
+console.log(whatsUrName());
```

Comment:
This technically is closure, but I'd like to see you challenge this a little more.